pom.xml

```diff
@@ -245,7 +245,7 @@
             </dependency>
             <dependency>
                 <groupId>io.undertow</groupId>
-                <artifactId>undertow-servlet-jakarta</artifactId>
+                <artifactId>undertow-servlet</artifactId>
```

Comment:
I think it would be good to also add a dependency for servlet API 6 (as Darran pointed out, currently, this gets brought in transitively by undertow-servlet).

Reply:
Explicit servlet-api dependency was added